### ISSUE TITLE
buildout annotate does not know of meta recipes

### ISSUE THREAD

**Issue by gotcha** [buildout/buildout#436]
This special case is not taken care of.

**Comment by idgserpro** [comment]
Indeed, I just came by this situation.

I was using buildout annotate for some pinned versions like it's said in a related issue https://github.com/buildout/buildout/issues/320. But I realized that in a `pylint` part, which uses `recipe = zc.recipe.egg`, which have a bunch of dependencies itself (like `six`, `astroid` and such), they aren't being outputted like I was expecting for a `bin/buildout annotate` command.

Is my assumption correct? Is this issue about this problem I just described?

**Comment by gotcha** [comment]
@idgserpro I do not think so. See http://www.buildout.org/en/latest/topics/meta-recipes.html to get more details on meta-recipes.
